modules/kms/variables.tf

```diff
@@ -15,16 +27,22 @@ variable "enable_key_rotation" {
   description = "Specifies whether key rotation is enabled"
 }
 
-variable "description" {
+variable "key_usage" {
   type        = string
-  default     = "Parameter Store KMS master key"
-  description = "The description of the key as viewed in AWS console"
+  default     = "ENCRYPT_DECRYPT"
+  description = "Specifies the intended use of the key. Valid values: `ENCRYPT_DECRYPT` or `SIGN_VERIFY`."
```

Comment:
Should we add a validation rule here or is this already done by the upstream module?